### ISSUE TITLE
Support `ph-no-capture` for Flutter widgets

### ISSUE THREAD

**Issue by marandaneto** [PostHog/posthog-flutter#137]
### Problem Statement

Android, iOS, and RN have a way of masking specific elements, but Flutter not yet

### Solution Brainstorm

https://posthog.com/docs/session-replay/privacy

Tag widgets with the `ph-no-capture` or a view modifier and mask them when taking screenshots

**Comment by marandaneto** [comment]
@thisames any ideas on how to do this?

I believe something like:

```
Text(
	"Capture",
).phNoCapture(true),
```

or specific [accessibility labels](https://appt.org/en/docs/flutter/samples/accessibility-label) - I'd avoid this if possible.

**Comment by thisames** [comment]
yes, I believe that with wrapper is the most easy way to do. But, I will analise which more make sense based in currently strucuture of masking and painting.
